qiime/split_libraries_lea_seq.py

```diff
@@ -194,6 +201,8 @@ def get_LEA_seq_consensus_seqs(fwd_read_f, rev_read_f,
                                  primer_mismatch_count,
                                  seq_too_short_count,
                                  total_seqs_kept)
+    if consensus_seq_lookup is None:
```

Comment:
If the consensus_seq_lookup doesn't end up being filled at all - if the user supplies very odd parameters, it's empty. I could catch that using this, when I tested it manually. I will add the unit test.